Source/Models/ChannelType.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Storage+ChannelType.swift
+//  VirgilMessenger
+//
+//  Created by Sergey Seroshtan on 30.03.2020.
+//  Copyright © 2020 VirgilSecurity. All rights reserved.
+//
+
+import Foundation
+
+public enum ChannelType: String, Codable {
+    case single
+    case group
```

Comment:
Maybe lets remove types we not support yet? Since e.g. it makes us add this case to every switch.